person.rb

```diff
@@ -1,16 +1,17 @@
 require './nameable'
 
 class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
   def initialize(age, name, parent_permission: true)
     super()
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
-  attr_accessor :name, :age
```

Comment:
- [ ] Great job adding rentals in the constructor, here we also need a method (add_rental) to add rentals, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.